src/System.Private.CoreLib/src/System/StubHelpers.cs

```diff
@@ -249,7 +249,7 @@ internal static unsafe IntPtr ConvertToNative(string strManaged, IntPtr pNativeB
                 {
                     // If not provided, allocate the buffer using SysAllocStringByteLen so
                     // that odd-sized strings will be handled as well.
-                    ptrToFirstChar = (byte*)Win32Native.SysAllocStringByteLen(null, lengthInBytes).ToPointer();
+                    ptrToFirstChar = (byte*)Interop.OleAut32.SysAllocStringByteLen(null, lengthInBytes).ToPointer();
```

Comment:
Nit: `.ToPointer()` should not be needed

Reply:
Deleted, here and elsewhere in the file.